packages/indexer-agent/src/agent.ts

```diff
@@ -582,7 +591,7 @@ class Agent {
 
     this.logger.debug(`Deployments to reconcile allocations for`, {
       number: deployments.length,
-      deployments: deployments.map(deployment => deployment.display),
+      deployments: deploymentIDSet(deployments),
```

Comment:
This will display the bytes32 representation of the deployment ID.  For consistency we may want to display the ipfs hash.